java/src/jmri/jmrix/can/cbus/CbusPreferences.java

```diff
@@ -28,9 +30,11 @@
     private boolean _saveRestoreEventTable = true;
     private int minimumNumBackupsToKeep = 10;
     private int bootWriteDelay = CbusNode.BOOT_PROG_TIMEOUT_FAST;
+    // Default to no programmers available. The p[rogrammer manager will validate
```

Comment:
`The p[rogrammer manager`
This seems to be a typo. No need to fix it now, you can fix it in a later PR if you want.